app/pods/components/agenda/agenda-detail/sidebar-item/template.hbs

```diff
@@ -6,7 +6,7 @@
   {{!-- Long term, I think we rather want this component to trigger an action when the "active" status toggles.
     With that, a higher-level component can decide if it wants to scroll this element in to view. Currently, because the active status
     is set right from the beginning and the component is rendered only once, scrolling on "did-insert" works just fine --}}
-  <div class="auk-u-flex" {{did-insert this.conditionallyScrollIntoView}} {{in-viewport onEnter=this.onEnter onExit=this.onExit}}>
+  <div class="auk-u-flex" {{did-insert this.conditionallyScrollIntoView}}>
     <span class="vlc-agenda-detail-sidebar-sub-item__numbering">
```

Comment:
Waarom is deze eruit ? dit was een fix dat de detailview ineens al deze data gaat ophalen voor alle agendapunten, ook al zijn die niet in view